pulpcore/cli/rpm/repository.py

```diff
@@ -67,8 +85,39 @@ def repository(ctx: click.Context, pulp_ctx: PulpContext, repo_type: str) -> Non
         raise NotImplementedError()
 
 
+package_options = [
+    click.option(
+        "--pulp-href",
```

Comment:
```suggestion
        "--package-href",
```

Reply:
I've changed it back to the `--package-href` as you suggested. However, I can't seem to get the `package_href` working for the json content. For now, I kept the `pulp_href` as the option keyword for the json. Is this way okay?